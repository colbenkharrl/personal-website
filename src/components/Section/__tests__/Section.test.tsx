import { render, screen } from '@testing-library/react';
import { Section } from '../';

it('Section matches snapshot (regression test)', () => {
  render(<Section title="Test title.">Text.</Section>);
  expect(screen.getByTestId('Section')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
