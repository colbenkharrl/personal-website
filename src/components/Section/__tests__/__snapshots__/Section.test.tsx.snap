// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section matches snapshot (regression test): Full component snapshot. 1`] = `
<section
  aria-labelledby=":r0:"
  class="md:border-l md:border-zinc-100 md:pl-6 md:dark:border-zinc-700/40"
  data-testid="Section"
>
  <div
    class="grid max-w-3xl grid-cols-1 items-baseline gap-y-8 md:grid-cols-4"
  >
    <h2
      class="text-sm font-semibold text-zinc-800 dark:text-zinc-100"
      id=":r0:"
    >
      Test title.
    </h2>
    <div
      class="md:col-span-3"
    >
      Text.
    </div>
  </div>
</section>
`;
